README.md

```diff
@@ -144,7 +143,7 @@ Some of the Trillian Go code is autogenerated from other files:
    [stringer](https://godoc.org/golang.org/x/tools/cmd/stringer) tool (`go get
    golang.org/x/tools/cmd/stringer`).
 
-Re-generating mock or protobuffer files is only needed if you're changing
+Re-generating mock or protobuffer files are only needed if you're changing
```

Comment:
Again, I think this was fine - _[the act of]_ regenerating <something> *is* only needed...